setup.py

```diff
@@ -110,14 +110,16 @@ def run(self):
 long_description = '''
 SymEngine is a standalone fast C++ symbolic manipulation library.
 Optional thin Python wrappers (SymEngine) allow easy usage from Python and
-integration with SymPy.'''
+integration with SymPy and Sage.'''
+
+exec(open('symengine/release.py').read())
```

Comment:
This is pretty ugly. I don't know if this can break things.

Here are some ideas how to do this: http://stackoverflow.com/questions/17583443/what-is-the-correct-way-to-share-package-version-with-setup-py-and-the-package

We should just set the version in `setup.py` as the only place, and then use it from the rest of the package. See e.g. how numpy does it:

https://github.com/numpy/numpy/blob/17146b2a873e835630ad58d00b73e97ba1fdb214/setup.py#L58
https://github.com/numpy/numpy/blob/17146b2a873e835630ad58d00b73e97ba1fdb214/numpy/__init__.py#L172
https://github.com/numpy/numpy/blob/17146b2a873e835630ad58d00b73e97ba1fdb214/numpy/version.py.in